packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkBridgeExtTest.kt

```diff
@@ -0,0 +1,361 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.datadog.tools.unit.GenericAssert.Companion.assertThat
+import com.datadog.tools.unit.keys
+import com.datadog.tools.unit.toReadableArray
+import com.datadog.tools.unit.toReadableMap
+import com.facebook.react.bridge.JavaOnlyArray
+import com.facebook.react.bridge.JavaOnlyMap
+import com.facebook.react.bridge.WritableArray
+import com.facebook.react.bridge.WritableMap
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(ExtendWith(MockitoExtension::class))
+@MockitoSettings(strictness = Strictness.LENIENT)
+internal class DdSdkBridgeExtTest {
+    // Default providers for toWritableArray and toWritableMap
+    private val createWritableMap = { JavaOnlyMap() }
+    private val createWritableArray = { JavaOnlyArray() }
+
+    @Test
+    fun `M call toWritableArray W all types of supported values`() {
```

Comment:
Normally for Android/Kotlin we follow the format: M - must; designates expected outcome in the assertions; W - when; designates the call made; {} - optional condition.

also very minor: we also tend to split test into `Given/When/Then` blocks for readability

```suggestion
    fun `M do a proper conversion W toWritableArray { all types of supported values }`() {
```